src/src/rx-serial/SerialGPS.h

```diff
@@ -0,0 +1,35 @@
+#include "SerialIO.h"
+
+typedef struct {
+    // Latitude in decimal degrees
+    uint32_t lat;
+    // Longitude in decimal degrees
+    uint32_t lon;
+    // Altitude in meters
+    uint32_t alt;
+    // Speed in km/h
+    uint32_t speed;
+    // Heading in degrees, positive. 0 is north.
+    uint32_t heading;
+    // Number of satellites
+    uint8_t satellites;
+} GpsData;
+
+class SerialGPS : public SerialIO {
+public:
+    explicit SerialGPS(Stream &out, Stream &in) : SerialIO(&out, &in)
+    {
```

Comment:
Any reason these braces are on single lines instead of just `{}` similar to `~SerialGPS() {}`?